test-files/cases/attribute/network/highway-2927-name-preservation-1/Expected.osm

```diff
@@ -507,24 +498,23 @@
         <nd ref="-81"/>
         <nd ref="-80"/>
         <nd ref="-35"/>
-        <tag k="surface" v="concrete"/>
-        <tag k="z_order" v="-999999"/>
-        <tag k="lit" v="yes"/>
+        <tag k="source" v="Yahoo;osm:line"/>
+        <tag k="oneway" v="no"/>
         <tag k="hoot:status" v="3"/>
-        <tag k="error:circular" v="15"/>
+        <tag k="source:ingest:datetime" v="2019-01-17T19:50:05.659Z"/>
+        <tag k="highway" v="pedestrian"/>
+        <tag k="license" v="This data is made available under the Open Database License: http://opendatacommons.org/licenses/odbl/1.0/."/>
         <tag k="name" v="Av. Oeste 0"/>
-        <tag k="source:datetime" v="2014-06-22T21:12:17.000Z;2015-08-11T16:12:27.000Z;2019-01-17T19:50:06Z"/>
-        <tag k="way_area" v="-999999"/>
-        <tag k="lanes" v="2"/>
         <tag k="uuid" v="{c708f207-166d-4d9c-936f-158a78dacb43}"/>
+        <tag k="lit" v="yes"/>
+        <tag k="source:datetime" v="2014-06-22T21:12:17.000Z;2015-08-11T16:12:27.000Z;2019-01-17T19:50:06Z"/>
+        <tag k="surface" v="concrete"/>
         <tag k="tags" v="\&quot;source:datetime\&quot;=&gt;\&quot;2018-10-30T13:05:51Z\\"/>
-        <tag k="attribution" v="osm"/>
-        <tag k="oneway" v="no"/>
-        <tag k="source:ingest:datetime" v="2019-01-17T19:50:05.659Z"/>
-        <tag k="license" v="This data is made available under the Open Database License: http://opendatacommons.org/licenses/odbl/1.0/."/>
+        <tag k="lanes" v="2"/>
+        <tag k="way_area" v="-999999"/>
         <tag k="osm_id" v="-999999"/>
-        <tag k="source" v="Yahoo;osm:line"/>
-        <tag k="highway" v="pedestrian"/>
+        <tag k="attribution" v="osm"/>
+        <tag k="z_order" v="-999999"/>
```

Comment:
This is an example of tag ordering issues.  All of the tags are identical just in a different order.